torch/csrc/jit/frontend/schema_type_parser.cpp

```diff
@@ -284,6 +284,7 @@ std::pair<TypePtr, c10::optional<AliasInfo>> SchemaTypeParser::parseType() {
   TypePtr value;
   c10::optional<AliasInfo> alias_info;
   // Tuple type
+  std::cout<<"In parseType::"<<L.cur().text()<<std::endl;
```

Comment:
What does this print out when trying to resolve the type?

Reply:
This one resolves to Tensor or __torch__. I did not see torch.classes.cuda.Stream.

Reply:
Hm, I made a mistake, I think we need to modify [script_type_parser.cpp](https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/frontend/script_type_parser.cpp#L170) since we're trying to parse types from a `Def`, not from source code directly.

I think we will still need these changes later when we enable serialization of code that uses CUDA streams, and at that time, we will be parsing the source directly.